tests/lib/external/snapd-testing-tools/utils/spread-manager

```diff
@@ -170,8 +154,24 @@ main() {
         show_help
         exit 1
     fi
-    "$action" "$params"
-    
+
+    while [ $# -gt 0 ]; do
```

Comment:
Since all cases have a `break`, do not we want an `if` instead of `while`?

Reply:
done